Control coverage/Feature/SubscriptionCore.md

```diff
@@ -438,18 +446,39 @@ Permanent access increase the risk of a malicious user getting that access and i
 ### Control Settings 
```

Comment:
Change the display name for this control to  **Do not grant permanent access for Subscription level roles**

Reply:
Fixed